src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +484,21 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    QList<QString> playlist_items;
+    int rows = m_pPlaylistTableModel->rowCount();
+    QList<QString> filenames;
```

Comment:
actually I'm going to have the dialog take a list of TrackPointers instead of locations.  In the future we may want to add more sophistication to the process, like make sure id3 tags are synced for each file, and even transcoding files on output.  CDJs don't support flac, so if the user (e.g. me :)) wants to use their tracks on a CDJ, they'll need to convert FLACs to AIFF or WAV.  By passing TrackPointers, we have a lot more information about the items being exported.